gpkit/nomials.py

```diff
@@ -240,7 +235,8 @@ def __str__(self, mult_symbol='*'):
         mstrs = []
         for c, exp in zip(self.cs, self.exps):
             varstrs = ['%s**%.2g' % (var, x) if x != 1 else "%s" % var
-                       for (var, x) in sorted(exp.items(), key=vkSortBy) if x != 0]
+                       for (var, x) in exp.items() if x != 0]
```

Comment:
The very next line sorts `varstrs` by string comparison. Is there some other type of sorting we want/need?

Previously we were running into issues because `vkSortBy` attempted to compare the `descr` of two `VarKey` objects, and when those keys have units, Pint tries to compare two `Quantity` objects with different units, resulting in a `DimensionalityError`.